posts/2017-03-15-faunadb-serverless-authentication.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Using Serverless Authentication Boilerplate with FaunaDB
+description: FaunaDB is a security aware database, so we pass user-specific access tokens to it via a custom authorizer.
+date: 2017-03-15
+thumbnail: https://cloud.githubusercontent.com/assets/20538501/23813615/4b53e4fc-05a5-11e7-8214-e34c2c02b949.png
+layout: Post
+authors:
+ - ChrisAnderson
+---
+
+We are proud to support our friends at Fauna as they announce [FaunaDB Serverless Cloud](https://fauna.com/product), a globally consistent distributed database. This is a guest post from Chris Anderson, Director of Developer Experience, showing how to connect FaunaDB's access control model with Serverless platform best practices.
```

Comment:
trying this: "...showing how FaunaDB's access control model can connect with best practices from the Serverless community."